Sources/MarathonCore/Install.swift

```diff
@@ -31,39 +31,67 @@ extension InstallError: PrintableError {
 // MARK: - Task
 
 internal class InstallTask: Task, Executable {
+    
     private typealias Error = InstallError
 
+    private var shouldForce: Bool {
+        return options.contains(.force) == true
+    }
+    
+    private var printVerbose: Bool {
+        return options.contains(.verbose) == true
+    }
+    
+    private let name: String?
+    private let installPath: String
+    private let options: [Option]
+    
+    init(arguments: [String], rootFolderPath: String, printer: Printer, options: [Option]) {
+        self.options = options
+        let element1 = arguments.element(at: 0)
+        let element2 = arguments.element(at: 1)
+        switch (element1 , element2) {
+        case (let value1?, let value2?):
+            self.name = value1
+            self.installPath = value2
+            super.init(rootFolderPath: rootFolderPath, printer: printer)
+        default:
+            self.name = arguments.first
+            let path = self.name ?? ""
+            self.installPath = "/usr/local/bin/\(path.lowercased())"
+            super.init(rootFolderPath: rootFolderPath, printer: printer)
+        }
+    }
+    
     func execute() throws {
-        guard let path = arguments.first else {
+        if let name = name {
+            try continueExecution(name)
+        } else {
             throw Error.missingPath
         }
-
-        let script = try scriptManager.script(atPath: path, allowRemote: true)
-        let installPath = makeInstallPath(for: script)
-
-        printer.reportProgress("Compiling script...")
+    }
+    
+    private func continueExecution(_ name: String) throws {
+        let scriptManager = try ScriptManager.assemble(with: rootPath, using: output)
+        
+        let script = try scriptManager.script(withName: name, allowRemote: true)
+        
+        output.progress("Compiling script...")
+        
         #if os(Linux)
```

Comment:
In keeping with the spirit of avoiding strings, I figured it would be cool to change the build commands throughout.